pkg/sentry/identity/spiffe.go

```diff
@@ -0,0 +1,35 @@
+package identity
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// CreateSPIFFEID returns a SPIFFE standard unique id for the given trust domain, namespace and appID.
+func CreateSPIFFEID(trustDomain, namespace, appID string) (string, error) {
+	if trustDomain == "" {
+		return "", errors.New("can't create spiffe id: trustDomain is empty")
```

Comment:
```suggestion
		return "", errors.New("can't create spiffe id: trust domain is empty")
```

for consistency with later errors?